guides/common/modules/proc_configuring-repositories-el.adoc

```diff
@@ -23,6 +23,16 @@ ifdef::katello[]
 ----
 # {package-manager} localinstall https://yum.theforeman.org/katello/{KatelloVersion}/katello/el{distribution-major-version}/x86_64/katello-repos-latest.rpm
 ----
+
+ifeval::["{distribution-major-version}" == "8"]
```

Comment:
this should only happen on CentOS, not RHEL.

Reply:
On testing this only renders for Centos 8